src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/interior_data_manager_impl.h

```diff
@@ -95,6 +102,7 @@ class InteriorDataManagerImpl : public InteriorDataManager {
    */
   void UnsubscribeFromInteriorVehicleData(const std::string& module_type);
 
+  void ClearOldSubscriptionsHostory();
```

Comment:
@LuxoftAKutsan typo, should be _ClearOldSubscriptionsHistory_

Reply:
06485260a2